functions_for_gyre.py

```diff
@@ -4,35 +4,36 @@
 import glob, os
 from . import read
 from . import my_python_functions as mypy
+import matplotlib.pyplot as plt
 import pandas
 ################################################################################
 def ledoux_splitting(frequencies, betas, Mstar, Rstar, omega=0, m=1):
     """
-    Calculate rotationally shifted frequencies. Aerts 2010, eq 3.357
+    Calculate rotationally shifted frequencies in a perturbative way. (See Aerts et al. (2010), eq 3.357)
     ------- Parameters -------
     frequencies, betas: numpy array of floats
-        frequency values (cyc/day) en beta values (eq 3.357, Aerts 2010)
+        frequency values (c/d) and beta values (eq 3.357, Aerts et al. (2010))
     Mstar, Rstar, omega: float
-        stellar mass (g), radius (cm) and rotation frequency in units of critical velocity (omega_crit)
+        stellar mass (g), radius (cm) and rotation frequency in units of critical velocity (omega_crit^-1)
     m: int
         azimuthal order
 
     ------- Returns -------
     shifted_freqs: numpy array of floats
-        frequency values as shifted by the ledoux splitting
+        pulsation frequency values, shifted by the Ledoux splitting
     """
 
-    G = 6.67428E-8 # gravitational constant (g^-1 cm^3 s^-2)
-    omega_crit = (1/(2*np.pi))*(8*G*Mstar/(27*Rstar**3))**0.5
-    omega_cycday = omega*omega_crit*86400 # rotation frequency in units of cyc/day
+    G = 6.67428E-8 # gravitational constant (g^-1 cm^3 s^-2) 
+    omega_crit = (1/(2*np.pi))*(8*G*Mstar/(27*Rstar**3))**0.5 # Roche critical rotation frequency (s^-1)
+    omega_cycday = omega*omega_crit*86400 # rotation frequency in units of c/d
 
-    shifted_freqs = frequencies-m*omega_cycday*(1-betas)
+    shifted_freqs = frequencies-(m*omega_cycday*(1-betas)) # shifted frequencies in units of c/d
     return shifted_freqs
 
 ################################################################################
-def calc_scanning_range(gyre_file_path, npg_min=-50, npg_max=-1, l=1, m=1, omega_rot=0.0):
+def calc_scanning_range(gyre_file_path, npg_min=-50, npg_max=-1, l=1, m=1, omega_rot=0.0, frame='inertial'):
     """
-    Calculate the frequency range for the sought radial orders of the g modes. (for non-rotating case)
+    Calculate the frequency range for the sought radial orders of the g modes. (non-rotating case)
```

Comment:
'(non-rotating case)' can be removed, since the functionality for rotation was added, but the documentation wasn't updated yet

Reply:
`Calculate the frequency range of the sought radial orders of the g modes.`